### PR TITLE
[Feat]: Homepage design

### PR DESCRIPTION
**Pull request by parthpandyappp**
Added the Landing/Home page design.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,189 @@
...
+
+.container {
   display: flex;
-  flex-direction: column;
+  gap: 1rem;
   align-items: center;
   justify-content: center;
-  color: white;
+  margin-top: 5rem;
+  width: 70%;
```
Comment (Logan1x) [inline src/App.css:93]: You might want to convert this to rem

**Review by Logan1x — APPROVED** [review]
Everything else looks good

**Comment by parthpandyappp** [comment]
Thanks @Logan1x for your valuable reviews. I'll look into it :+1:

**Review by sandeepkumardev-zz — APPROVED** [review]
looks good for me, clean code 👍

**Pull request merged**